File: Codeforces/Codeforces_678_2020/codeforces.01.cpp
#include<bits/stdc++.h>
#define ll long long int
#define endl "\n"
using namespace std;

int main()
{   
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    ll test;vector<ll> arr;
    cin >> test;
    while(test--)
    {
        ll sum = 0, i, j, k, n, m, temp, count = 0, ans = 0;
        cin >> n >> m;

        ll arr[n];

        for(i = 0; i < n; i++)
        {
            cin >> arr[i];  
            sum += arr[i];
        }

        if(sum == m)
        {
            cout << "YES" << endl;
        }
        else    
            cout << "NO" << endl;
    }

return 0;
}

File: Codeforces/Codeforces_Edu_96_2020/codeforces.01.py
try:
    test = int(input())
    for _ in range(test):
        n = int(input())
        ans = n // 3
        if n == 1 or n == 2 or n == 4:
            print("-1")
        else:
            if n == 5:
                print("0 1 0")
            elif n == 7:
                print("0 0 1") 
            elif n % 3 == 2:
                print(str(ans - 1) + " 1 0")
            elif n % 3 == 1:
                print(str(ans - 2) + " 0 1")
            else:
                print(str(ans) + " 0 0")
except:
    pass